### PR TITLE
Add cream-swap strategy

### PR DESCRIPTION
**Pull request by bun919tw**
Add cream-swap strategy.

Changes proposed in this pull request:
- Add cream-swap strategy.
- Fix strategy format in test.

### CODE REVIEW

File: src/strategies/cream-swap/index.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest } from '../../utils';
+
+export const author = 'creamdev';
```
Comment (bonustrack) [inline src/strategies/cream-swap/index.ts:4]: This must be a valid GitHub username

Comment (bun919tw) [reply src/strategies/cream-swap/index.ts]: Updated!

**Comment by bun919tw** [comment]
@bonustrack Hi, could you take a look again? Thanks a lot :)

File: src/strategies/cream-swap/index.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest } from '../../utils';
+
+export const author = 'CreamFinanceDev';
```
Comment (bonustrack) [inline src/strategies/cream-swap/index.ts:4]: This must be your github username

Comment (bun919tw) [reply src/strategies/cream-swap/index.ts]: updated!

**Pull request merged**